### PR TITLE
users/nir/gmp/code_refactoring

### PR DESCRIPTION
**Pull request by nirgalili**
PR goal:
- code refactoring and git commands practice

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # Gold Mining Prediction
-Prepare a prototype of a machine learning model for Zyfra. The model should predict the amount of gold recovered from gold ore. 
+Prepare a prototype of a machine learning model for Zyfra. The model should predict the amount of gold recovered from gold ore.
+
+## Running Intsructions
```
Comment (MayaGalili) [inline README.md]: Fix typo

Comment (MayaGalili) [inline README.md]: add the instructions

Comment (nirgalili) [reply README.md]: done

**Review by MayaGalili — APPROVED** [review]
Great job!!!!

**Pull request merged**